src/axom/slam/policies/IndirectionPolicies.hpp

```diff
@@ -272,6 +277,66 @@ struct CoreArrayIndirection
   IndirectionBufferType* m_vecBuf;
 };
 
+/**
+ * \brief A policy class for sets with axom::ArrayView-based indirection
+ */
+template <typename PositionType, typename ElementType>
+struct ViewIndirection
+{
+  using IndirectionResult = ElementType&;
+  using ConstIndirectionResult = const ElementType&;
+
+  using VectorType = axom::ArrayView<ElementType>;
+  using IndirectionBufferType = VectorType;
+  using IndirectionPtrType = IndirectionBufferType;
+
+  ViewIndirection(IndirectionBufferType buf = {}) : m_vecBuf(buf) { }
+
+  IndirectionBufferType data() const { return m_vecBuf; }
+
+  inline ConstIndirectionResult indirection(PositionType pos) const
+  {
+    SLIC_ASSERT_MSG(hasIndirection(),
+                    "SLAM::Set:CoreArrayIndirection -- Tried to dereference "
```

Comment:
Please update this message to reflect the name of the class; 
(currently `ViewIndirection`, perhaps `ArrayViewIndirection`)